models/events.sql

```diff
@@ -241,8 +246,8 @@ where
     event_type is not null and
     event_time >= '{{ var("fullstory_min_event_time") }}'
     {% if is_incremental() %}
-        -- we can't use the max event_time because event_time is specified by the client. We cannot guarantee
-        -- that it is accurate. Instead, we will use the current timestamp, and look back a configurable
-        -- distance for updates.
-        and cast(event_time as timestamp) >= current_timestamp - {{ var("fullstory_incremental_interval") }}
+    -- we can't use the max event_time because event_time is specified by the client. We cannot guarantee
```

Comment:
I don't quite understand this comment, is it saying that new sync can contain events with older event_time?

Reply:
can we use `processed_time` instead or something?

Reply:
It's possible that _new_ data is late arriving and we don't want to miss it, so we're basically just configuring the look back window to a static value (8 days).

We can't use processed time unfortunately because the source table is partitioned on `event_time`; so we need to use `event_time` to have any sort of efficiency.